railties/test/generators/shared_generator_tests.rb

```diff
@@ -264,7 +265,7 @@ def test_generator_does_not_generate_active_storage_contents_if_skip_active_reco
     assert_file "#{application_path}/config/application.rb", /#\s+require\s+["']active_storage\/engine["']/
 
     assert_file "#{application_path}/app/javascript/packs/application.js" do |content|
-      assert_no_match(/^require\("@rails\/activestorage"\)\.start\(\)/, content)
+      assert_no_match(/^import * as ActiveStorage from "@rails\/activestorage"\nActiveStorage.start\(\)/, content)
```

Comment:
I think the `*` should be escaped (`\*`), right? 

And, I think the `ActiveStorage.start()` test should be a separate assertion. I don't think it will fail correctly as-is since it doesn't immediately follow the import line, but the asserting expects it to. https://github.com/rails/rails/blob/8caacd0942d6aaecfc23ac8e7d81dbce45a022bd/railties/lib/rails/generators/rails/app/templates/app/javascript/packs/application.js.tt#L10-L23

Reply:
Since it's easy for such an assertion to accidentally pass, what about something less specific?

```suggestion
      assert_no_match(/ActiveStorage/, content)
```

Reply:
Case-insensitive match would cover the `import` and `start()` statement.
```suggestion
      assert_no_match(/activestorage/i, content)
```

Reply:
Good call! I agree this works much better. Updated to `assert_no_match(/activestorage/i, content)`